### DISCUSSION TITLE
Front end templates for TALL (Tailwind,Alpine,Laravel,Livewire) stack to point me to

### DISCUSSION THREAD

**Discussion by CryptedSnow** [livewire/livewire#8847] [1. Help]
I am not very good in front end development, i understand just the back end. I would like if exist templates for help me in my applications. If some one point me to these templates. I am using [TALL](https://tallstack.dev/) stack.

**Comment by danie-ramdhani** [comment]
You can try [Mary UI](https://mary-ui.com) or you can support mr caleb by purchasing [Flux UI](https://fluxui.dev)

**Reply by CryptedSnow** [reply]
@danie-ramdhani is there any template done to dashboard for example made using TALL stack?

**Comment by PhiloNL** [comment]
You can use Jetstream, it has a Livewire stack:
https://jetstream.laravel.com/introduction.html#livewire-blade
